test/utils.cpp

```diff
@@ -399,13 +399,6 @@ TEST_CASE("quote_for_stfl() adds a \'>\' after every \'<\'", "[utils]")
 }
 
 
-TEST_CASE("quote()", "[utils]")
```

Comment:
No need to remove the test: even though we have Rust tests now, this one will go through FFI and check that we didn't introduce any bugs there. Please bring it back.